website/docs/user-guide/concepts/environment-variables.md

```diff
@@ -0,0 +1,68 @@
+# Environment Variables
+
+In the [Analyzer](https://oss-review-toolkit.org/ort/#analyzer)
+step of the [OSS Review Toolkit](https://oss-review-toolkit.org/ort/) pipeline,
+dependencies of your project are identified, and
+information about them is gathered, such as declared license, authors and source code location.
+
+In detail, package managers perform this task. As in most cases it is **not** appropriate to configure these package
+managers by files that are located in the source code repository of your project, typically environment variables
+are used by CI/CD pipelines and also by the OSS Review Toolkit pipeline.
```

Comment:
> As in most cases it is **not** appropriate to configure these package managers by files that are located in the source code repository...

That part is unclear to me, why is it not appropriate in "most cases"? It's also a bit weird because the solution below is to add a file to the repository.